www/kore07/cv02/index.php

```diff
@@ -0,0 +1,37 @@
+<?php require './Person.php' ?>
+<?php
+
+$persons = [
+    new Person("Kortoshkina", "Katerina", 1997, "student", "VSE", "nám. W. Churchilla", 1938, 4, "Praha", "778888999", "ekortoshkina@seznam.cz", "www.web.cz", true, "https://upload.wikimedia.org/wikipedia/commons/1/1e/RPC-JP_Logo.png"),
+    new Person("Pelevin", "Nikita", 1995, "student", "VSCHT", "Technicka", 24, 5, "Praha", "999555666", "npelevin@seznam.cz", "www.business.cz", true, "https://upload.wikimedia.org/wikipedia/commons/1/1e/RPC-JP_Logo.png"),

```

Comment:
kdyz mate moc dlouhy radek tak muzete to zalomit - kazdy argument na jeden radek